packages/account/src/Sections/Security/AccountLimits/account-limits.tsx

```diff
@@ -1,8 +1,9 @@
 import AccountLimits from 'Components/account-limits/account-limits.jsx';
 import 'Components/account-limits/account-limits.scss';
 import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
 
-export default connect(({ client, common, ui }) => ({
+export default connect(({ client, common, ui }: RootStore) => ({
```

Comment:
@yauheni-deriv The `RootStore` type is just next level `any` 😅
You can use the `TRootStore` type from `@deriv/stores` instead to get type-safety 🙇🏻